packages/salesforce-adapter/src/metadata_deploy.ts

```diff
@@ -94,22 +94,24 @@ type MetadataId = {
   fullName: string
 }
 
-const getUnFoundDeleteName = (message: DeployMessage): MetadataId | undefined => {
+const getUnFoundDeleteName = (message: DeployMessage,
+  deletionsPackageName: string): MetadataId | undefined => {
```

Comment:
line break style nit
```suggestion
const getUnFoundDeleteName = (
  message: DeployMessage,
  deletionsPackageName: string,
): MetadataId | undefined => {
```
or
```suggestion
const getUnFoundDeleteName = (
  message: DeployMessage, deletionsPackageName: string,
): MetadataId | undefined => {
```